tilavarauspalvelu/integrations/email/template_context/application.py

```diff
@@ -134,7 +134,13 @@ def get_context_for_application_section_cancelled(
             "email_recipient_name": application_section.application.applicant,
             "cancel_reason": get_attr_by_language(reservation.cancel_reason, "reason", language),
             **params_for_application_section_info(application_section=application_section, language=language),
+            **get_contex_for_seasonal_reservation_check_details_url(
+                language=language,
+                application_section=application_section,
+            ),
         }
+    else:
+        data.update(get_contex_for_seasonal_reservation_check_details_url(language=language))
```

Comment:
Since this doesn't get the section, it creates a different link when using e.g. the email tester in django admin. This might be confusing, so I would make this a different function that can create the same link even without the section itself.